### PR TITLE
Update Fields - Calculated Fields - Overview of user document

### PR DESCRIPTION
**Pull request by recca5p**
…erview

### CODE REVIEW

File: docs/user/fields/calculated/images/new-calculated-field-dialog.png
Comment (hainv-ohio) [inline docs/user/fields/calculated/images/new-calculated-field-dialog.png:1]: It's quite small to see the text. Please zoom in the input field, don't need to show the whole UI below it.

File: docs/user/fields/calculated/images/post-calculated-field-new-calculated-field-dialog.png
Comment (hainv-ohio) [inline docs/user/fields/calculated/images/post-calculated-field-new-calculated-field-dialog.png:1]: same comment, pls zoom in

**Review by hainv-ohio — CHANGES_REQUESTED** [review]

File: docs/user/fields/calculated/images/post-calculated-field-new-calculated-field-dialog.png
Comment (recca5p) [reply docs/user/fields/calculated/images/post-calculated-field-new-calculated-field-dialog.png:1]: I updated it, please re-review it

File: docs/user/fields/calculated/images/new-calculated-field-dialog.png
Comment (recca5p) [reply docs/user/fields/calculated/images/new-calculated-field-dialog.png:1]: I updated it, please re-review it

**Review by hainv-ohio — APPROVED** [review]

**Pull request merged**